src/backend/core/models.py

```diff
@@ -452,6 +454,34 @@ def get_abilities(self, user):
         }
 
 
+class Invitation(BaseModel):
+    """User invitation to teams."""
+
+    email = models.EmailField(_("email address"), null=False, blank=False)
+    team = models.ForeignKey(
+        Team,
+        on_delete=models.CASCADE,
+        related_name="invitations",
+    )
+    role = models.CharField(
+        max_length=20, choices=RoleChoices.choices, default=RoleChoices.MEMBER
+    )
+    issuer = models.ForeignKey(
+        User, on_delete=models.SET_NULL, related_name="invitations", null=True
+    )
+
+    def __str__(self):
+        return f"{self.email} invited to {self.team}"
```

Comment:
should we translate it?

Reply:
What do you mean ? Won't we translate it in crowdin ?